src/auto/pixbuf.rs

```diff
@@ -0,0 +1,531 @@
+            let ret = from_glib_none(ffi::gdk_pixbuf_get_file_info(filename.to_glib_none().0, &mut width, &mut height));
+            (ret, width, height)
+        }
+    }
+
+    #[cfg(any(feature = "v2_32", feature = "dox"))]
+    pub fn get_file_info_async<'a, P: Into<Option<&'a gio::Cancellable>>, Q: Fn(Result<(i32, i32), Error>) + Send + Sync + 'static>(filename: &str, cancellable: P, callback: Q) {
+        let cancellable = cancellable.into();
+        let cancellable = cancellable.to_glib_none();
+        let user_data: Box<Box<Fn(Result<(i32, i32), Error>) + Send + Sync + 'static>> = Box::new(Box::new(callback));
+        extern "C" fn get_file_info_async_trampoline(_source_object: *mut gobject_ffi::GObject, res: *mut gio_ffi::GAsyncResult, user_data: glib_ffi::gpointer)
+        {
+            callback_guard!();
+            unsafe {
+                let mut error = ptr::null_mut();
+                let mut width = mem::uninitialized();
+                let mut height = mem::uninitialized();
+                let _ = ffi::gdk_pixbuf_get_file_info_finish(res, &mut width, &mut height, &mut error);
+                let result = if error.is_null() { Ok((width, height)) } else { Err(from_glib_full(error)) };
+                let callback: &&(Fn(Result<(i32, i32), Error>) + Send + Sync + 'static) = transmute(user_data);
+                callback(result);
+            }
+        }
+        let callback = get_file_info_async_trampoline;
+        unsafe {
+            ffi::gdk_pixbuf_get_file_info_async(filename.to_glib_none().0, cancellable.0, Some(callback), Box::into_raw(user_data) as *mut _);
+        }
+    }
+
+    pub fn get_formats() -> Vec<PixbufFormat> {
+        unsafe {
+            FromGlibPtrContainer::from_glib_container(ffi::gdk_pixbuf_get_formats())
+        }
+    }
+
+    //pub fn new_from_stream_async<'a, P: IsA</*Ignored*/gio::InputStream>, Q: Into<Option<&'a gio::Cancellable>>, R: Fn(Result<(), Error>) + Send + Sync + 'static>(stream: &P, cancellable: Q, callback: R) {
+    //    unsafe { TODO: call ffi::gdk_pixbuf_new_from_stream_async() }
+    //}
+
+    //pub fn new_from_stream_at_scale_async<'a, P: IsA</*Ignored*/gio::InputStream>, Q: Into<Option<&'a gio::Cancellable>>, R: /*Ignored*/gio::AsyncReadyCallback>(stream: &P, width: i32, height: i32, preserve_aspect_ratio: bool, cancellable: Q, callback: R) {
+    //    unsafe { TODO: call ffi::gdk_pixbuf_new_from_stream_at_scale_async() }
+    //}
+}
+
+pub trait PixbufExt {
+    fn apply_embedded_orientation(&self) -> Option<Pixbuf>;
+
+    #[cfg(any(feature = "v2_36", feature = "dox"))]
+    fn copy_options(&self, dest_pixbuf: &Pixbuf) -> bool;
+
+    fn fill(&self, pixel: u32);
+
+    fn get_option(&self, key: &str) -> Option<String>;
+
+    //#[cfg(any(feature = "v2_32", feature = "dox"))]
+    //fn get_options(&self) -> /*Unknown conversion*//*Unimplemented*/HashTable TypeId { ns_id: 0, id: 28 }/TypeId { ns_id: 0, id: 28 };
```

Comment:
Not sure why this is not generated. It's a `HashMap<String, String>` that would be needed here, which we cover in glib's translate.rs

Reply:
Added issue to remember this https://github.com/gtk-rs/gir/issues/507